internal/cache/clockpro.go

```diff
@@ -447,19 +463,15 @@ func (c *shard) runHandHot() {
 			c.sizeCold += e.size
 		}
 	}
-
-	c.handHot = c.handHot.next()
+	// Drag handTest if handHot has passed handTest.
```

Comment:
Dragging is better than pushing because the result is same but we don't need to nil check.